charts/virtual-kubelet/values.yaml

```diff
@@ -42,7 +42,24 @@ providers:
       subnetCidr: 
       clusterCidr: 
       kubeDnsIp: 
-
+  aws:
+    targetFargate: true
```

Comment:
is there a scenario that aws is enabled but not target fargate?

Reply:
Mhh.. good point. I was trying to make the AWS/Fargate configuration similar to the Azure/AKS one, see `providers.azure.targetAKS` -- but I think I might have misunderstood the meaning of this flag.